storage/storage_tests/create_country_info_getter.hpp

```diff
@@ -0,0 +1,15 @@
+#pragma once
+
+#include "geometry/mercator.hpp"
+
+#include "std/unique_ptr.hpp"
+
+namespace storage
+{
+class CountryInfoGetter;
+
+unique_ptr<CountryInfoGetter> CreateCountryInfoGetter();
```

Comment:
It does not look good in my opinion. Just compare:

```
unique_ptr<storage::CountryInfoGetter> CreateCountryInfoGetter(bool isSingleMwm)		
 {		
   Platform & platform = GetPlatform();		
   string const packedPolygons = isSingleMwm ? PACKED_POLYGONS_MIGRATE_FILE : PACKED_POLYGONS_FILE;		
   string const countryTxt = isSingleMwm ? COUNTRIES_MIGRATE_FILE : COUNTRIES_FILE;		
   return make_unique<storage::CountryInfoReader>(platform.GetReader(packedPolygons),		
                                                  platform.GetReader(countryTxt));		
 }
```
and
```
unique_ptr<CountryInfoGetter> CreateCountryInfoGetter()
{
  Platform & platform = GetPlatform();
  return unique_ptr<CountryInfoGetter>(new CountryInfoReader(platform.GetReader(PACKED_POLYGONS_FILE),
                                                             platform.GetReader(COUNTRIES_FILE)));
}

unique_ptr<storage::CountryInfoGetter> CreateCountryInfoGetterMigrate()
{
  Platform & platform = GetPlatform();
  return unique_ptr<CountryInfoGetter>(new CountryInfoReader(platform.GetReader(PACKED_POLYGONS_MIGRATE_FILE),
                                                             platform.GetReader(COUNTRIES_MIGRATE_FILE)));
}
```
What is better with you?